src/SCREENS.ts

```diff
@@ -30,14 +49,162 @@ const SCREENS = {
             ADDRESS: 'Settings_Card_Get_Physical_Address',
             CONFIRM: 'Settings_Card_Get_Physical_Confirm',
         },
+        WALLET_TRANSFER_BALANCE: 'Settings_Wallet_Transfer_Balance',
+        WALLET_CHOOSE_TRANSFER_ACCOUNT: 'Settings_Wallet_Choose_Transfer_Account',
+        WALLET_ENABLE_PAYMENTS: 'Settings_Wallet_EnablePayments',
+        WALLET_CARD_ACTIVATE: 'Settings_Wallet_Card_Activate',
+        WALLET_REPORT_VIRTUAL_CARD_FRAUD: 'Settings_Wallet_ReportVirtualCardFraud',
```

Comment:
I am a little bit confused about pattern for these values. Somewhere the third position uses `UpperCamelCase`, other `UPPER_SNAKE_CASE`. So...?
```suggestion
        WALLET_REPORT_VIRTUAL_CARD_FRAUD: 'Settings_Wallet_Report_Virtual_Card_Fraud',
```

Reply:
@blazejkustra Since you have [related PR](https://github.com/Expensify/App/pull/32430/files) as well, what do you think? It would be good to keep our updates aligned to prevent many conflicts or maybe even hold one PR for another.

Reply:
I don't think changing these values bring any value, in the end we will use `SCREENS.<name>` in the app, so imo let's not edit at all.